crates/voicevox_core/src/__internal.rs

```diff
@@ -0,0 +1,9 @@
+// FIXME: 要議論: https://github.com/VOICEVOX/voicevox_core/issues/595
```

Comment:
`__internal.rs`がRustで一般的でなければ、このファイルの目的をコメント書いておくと意図が伝わりやすいかもです。
こんな感じ･････？
（そこまで強い意見じゃなく、must/should/want/canのwantくらいのモチベーションです。）

```suggestion
// VOICEVOX CORE内のラッパー向けの実装
// FIXME: 要議論: https://github.com/VOICEVOX/voicevox_core/issues/595
```

Reply:
docstringにしました (`#[doc(hidden)]`なアイテムは補完には出なくなりますが、一度記述されればRust Analyzerで普通に見ることができるので)
[`1020465` (#589)](https://github.com/VOICEVOX/voicevox_core/pull/589/commits/1020465c5a8e2747f7bf9687e1097a7bc19c1704)

![image](https://github.com/VOICEVOX/voicevox_core/assets/14125495/2242d30f-f873-47d8-94a1-81b9d3ecb1fa)